syntax = "proto3";

package proto;


message PpgRequest {
    string id = 1;
    int32 anoi = 2;
    int32 anof = 3;
}

message PpgJson {
    string nome = 1;
    string json = 2;
}

message PpgResponse {
  repeated PpgJson item = 1;
}